Added roboto slab font

Blazevarjo opened the pull request.

Blazevarjo committed:
Added robot slab font

kamil-matula reviewed (changes requested):
DetailsAppBar, PokemonStats, StatBar and PokemonMovesList's header look different than in Figma:

<p>
<img src="https://user-images.githubusercontent.com/43967269/120085208-7b5a9380-c0d6-11eb-8122-6735245a0cc7.png" width="400"/>
<img src="https://user-images.githubusercontent.com/43967269/120085442-6ed73a80-c0d8-11eb-94bb-ee0c09b1b55c.png" width="400"/>
</p>

<p>
<img src="https://user-images.githubusercontent.com/43967269/120085465-9d551580-c0d8-11eb-97bf-edaac4c0c261.png" width="400"/>
<img src="https://user-images.githubusercontent.com/43967269/120085470-a5ad5080-c0d8-11eb-9607-0e29c2122e92.png" width="400"/>
</p>

<p>
<img src="https://user-images.githubusercontent.com/43967269/120085473-b231a900-c0d8-11eb-962f-b219e95be9f3.png" width="400"/>
<img src="https://user-images.githubusercontent.com/43967269/120085482-c07fc500-c0d8-11eb-8b54-24e93892a3a9.png" width="400"/>
</p>

Make sure that title in DetailsAppBar is capitalized and that in the rest three components Roboto Slab is used. You can check how to fix it in #21.

kamil-matula committed:
Fixed non-regular fonts (manually)

Added manually:
- regular Roboto Slab font to PokemonStats and StatBar,
- medium Roboto Slab font to PokemonMovesList's header,
- bold Roboto Slab font to Sorting Menu's headers.

Also capitalized DetailsAppBar's "Pokedex" caption.
